src/components/NetworkAlert/NetworkAlert.tsx

```diff
@@ -1,200 +1,73 @@
-    [ChainId.BNB]:
-      'radial-gradient(182.71% 150.59% at 2.81% 7.69%, rgba(240, 185, 11, 0.16) 0%, rgba(255, 168, 0, 0.16) 100%)',
-    [ChainId.OPTIMISM]:
-      'radial-gradient(92% 105% at 50% 7%, rgba(255, 58, 212, 0.04) 0%, rgba(255, 255, 255, 0.03) 100%),radial-gradient(100% 97% at 0% 12%, rgba(235, 0, 255, 0.1) 0%, rgba(243, 19, 19, 0.1) 100%), hsla(0, 0%, 100%, 0.1)',
-    [ChainId.OPTIMISM_GOERLI]:
-      'radial-gradient(92% 105% at 50% 7%, rgba(255, 58, 212, 0.04) 0%, rgba(255, 255, 255, 0.03) 100%),radial-gradient(100% 97% at 0% 12%, rgba(235, 0, 255, 0.1) 0%, rgba(243, 19, 19, 0.1) 100%), hsla(0, 0%, 100%, 0.1)',
-    [ChainId.ARBITRUM_ONE]:
-      'radial-gradient(285% 8200% at 30% 50%, rgba(40, 160, 240, 0.1) 0%, rgba(219, 255, 0, 0) 100%),radial-gradient(circle at top left, hsla(206, 50%, 75%, 0.01), hsla(215, 79%, 51%, 0.12)), hsla(0, 0%, 100%, 0.1)',
-    [ChainId.ARBITRUM_GOERLI]:
-      'radial-gradient(285% 8200% at 30% 50%, rgba(40, 160, 240, 0.1) 0%, rgba(219, 255, 0, 0) 100%),radial-gradient(circle at top left, hsla(206, 50%, 75%, 0.01), hsla(215, 79%, 51%, 0.12)), hsla(0, 0%, 100%, 0.1)',
-    [ChainId.AVALANCHE]:
-      'radial-gradient(92% 105% at 50% 7%, rgba(255, 58, 212, 0.04) 0%, rgba(255, 255, 255, 0.03) 100%),radial-gradient(100% 97% at 0% 12%, rgba(235, 0, 255, 0.1) 0%, rgba(243, 19, 19, 0.1) 100%), hsla(0, 0%, 100%, 0.1)',
-    [ChainId.BASE]:
-      'radial-gradient(100% 100% at 50% 0%, rgba(0, 82, 255, 0.20) 0%, rgba(0, 82, 255, 0.08) 40.0%, rgba(252, 255, 82, 0.00) 100%), rgb(255, 255, 255)',
-  },
-}
+  border-radius: 8px;
+  color: white;
+  display: flex;
+  font-size: 16px;
+  justify-content: space-between;
+  padding: 12px 18px 12px 12px;
+  text-decoration: none !important;
+  width: 100%;
 
-const ContentWrapper = styled.div<{ chainId: NetworkAlertChains; darkMode: boolean; logoUrl: string }>`
-  background: ${({ chainId, darkMode }) => BG_COLORS_BY_DARK_MODE_AND_CHAIN_ID[darkMode ? 'dark' : 'light'][chainId]};
   border-radius: 20px;
   display: flex;
   flex-direction: row;
   overflow: hidden;
   position: relative;
   width: 100%;
 
-  :before {
-    background-image: url(${({ logoUrl }) => logoUrl});
-    background-repeat: no-repeat;
-    background-size: 300px;
-    content: '';
-    height: 300px;
-    opacity: 0.1;
-    position: absolute;
-    transform: rotate(25deg) translate(-90px, -40px);
-    width: 300px;
-    pointer-events: none;
-  }
-`
-const Header = styled.h2`
-  font-weight: 535;
-  font-size: 16px;
-  margin: 0;
+  margin-top: 16px;
 `
 
-const LinkOutToBridge = styled(ExternalLink)`
-  align-items: center;
-  border-radius: 8px;
-  color: white;
-  display: flex;
-  font-size: 16px;
-  justify-content: space-between;
-  padding: 6px 8px;
-  text-decoration: none !important;
-  width: 100%;
+const TitleText = styled(ThemedText.BodyPrimary).attrs({ fontWeight: 535 })<{ color: string }>`
```

Comment:
why did you use the `attrs` pattern here over adding the fontWeight 535 as a style below?

Reply:
nit: `$color` for var name of a css style

Reply:
setting it in the style didn't work?! I've noticed I've had to manually set it in attr's lately. I can investigate why separately